[2025-04-14T17:36:53.963+0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-14T17:36:53.973+0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: executing_python_task.task_a scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-04-14T17:36:53.985+0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: executing_python_task.task_a scheduled__2025-04-13T00:00:00+00:00 [queued]>
[2025-04-14T17:36:53.985+0300] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-14T17:36:54.004+0300] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): task_a> on 2025-04-13 00:00:00+00:00
[2025-04-14T17:36:54.008+0300] {standard_task_runner.py:72} INFO - Started process 4194 to run task
[2025-04-14T17:36:54.031+0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'executing_python_task', 'task_a', 'scheduled__2025-04-13T00:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/execute_python_operator.py', '--cfg-path', '/tmp/tmposj2wim1']
[2025-04-14T17:36:54.039+0300] {standard_task_runner.py:105} INFO - Job 40: Subtask task_a
[2025-04-14T17:36:54.108+0300] {task_command.py:467} INFO - Running <TaskInstance: executing_python_task.task_a scheduled__2025-04-13T00:00:00+00:00 [running]> on host DESKTOP-1SBO1JU.
[2025-04-14T17:36:54.194+0300] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='tasneemahmed' AIRFLOW_CTX_DAG_ID='executing_python_task' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2025-04-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-13T00:00:00+00:00'
[2025-04-14T17:36:54.195+0300] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-14T17:36:54.218+0300] {logging_mixin.py:190} INFO - Hello from Python function!
[2025-04-14T17:36:54.219+0300] {python.py:240} INFO - Done. Returned value was: None
[2025-04-14T17:36:54.227+0300] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-14T17:36:54.228+0300] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=executing_python_task, task_id=task_a, run_id=scheduled__2025-04-13T00:00:00+00:00, execution_date=20250413T000000, start_date=20250414T143653, end_date=20250414T143654
[2025-04-14T17:36:54.290+0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-14T17:36:54.301+0300] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-14T17:36:54.302+0300] {local_task_job_runner.py:245} INFO - ::endgroup::
